### DISCUSSION TITLE
Pk-sim updates bug

### DISCUSSION THREAD

**Discussion by Shahejona** [Open-Systems-Pharmacology/Forum#1645] [Questions & Problems] [answered]
Hi Everyone, 

I recently noticed that the built-in values of my validated model did not match the built-in values of the DDI model. I started working on this model with v11.0, then updated PK-sim as soon as I could to v11.1 and afterwards to v11.2.
I have noticed that the interstitial/intrinsic permeabilities are different in my simulations causing problems in the model. Furthermore I am not sure whether more (built-in) values were changed in the updates

Q1: is there a neat fix to this bug without manually changing the values?
Q2: how can I be sure that ONLY these values differ between the versions?

Any input is greatly appreciated, 

Kind regards from the Netherlands
Jonathan Shaheen

**Comment by StephanSchaller** [comment]
Hi Jonathan, can you clarify/specify what you mean by "the built-in values of my validated model did not match the built-in values of the DDI model"?
What is your model, and what is the "DDI Model"?

You can also check the [release notes](https://github.com/Open-Systems-Pharmacology/Suite/releases/tag/v11.2) for the different versions to get an idea if updated features or fixed bugs would affect your model.

Best,
Stephan

**Reply by Shahejona** [reply]
Thank you for your response Stephan, 

What I mean with the built-in values of pk-sim are the individual organ permeabilities (intracellular -> interstitial). Our model was developed and optimized in human via IV, then PO administration (v11.0). Now that I am moving forward with a new simulation (v11.2) with DDI of rifampicin I see that the "organ permeabilities from intracellular to interstitial space" are different (simulation > parameters > advanced > permeability > P (intracellular -> interstitial). 

These values were set as 0.09 cm/min for all organs in previous versions. However, newly made simulations with v11.2 show organ permeability values ranging between 0.02 and 0.05. 
(screenshots,
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/148430478/79b18c88-c5d0-4a96-916c-48d8453d0465)
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/148430478/9190ad37-66dd-4a4d-ab68-530fc6824e62)
)


Therefore my two abovementioned questions, do you have any ideas /suggestions about it?


I will go through the notes to check up on the changes (so far, I have found this https://github.com/Open-Systems-Pharmacology/PK-Sim/wiki/Intestinal-Permeability-Calibration)

Best regards,
Jonathan

**Comment by Yuri05** [comment, marked as answer]
Cellular permeability calculation **did not change between versions**.
Looking at your screenshots, it seems that 
a) either you are using **different calculation methods** for cellular permeability (e.g. "Standard" in 11.0 and "Schmitt" in 11.2) 
b) or you are working **with different compound properties** in 11.0 and 11.2 (e.g. added/modified pKa values)

**Reply by Shahejona** [reply]
HI Yuri, 

Indeed different calculation methods were used. The new DDI simulations did not have the same calculation methods as the IV and PO model. 

Thank you very much for you reply.
